[2025-02-17T19:01:13.604+0000] {processor.py:186} INFO - Started process (PID=5719) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:01:13.605+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:01:13.607+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:01:13.606+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:01:13.609+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:01:13.608+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:01:13.609+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:01:13.609+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:01:13.609+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:01:13.615+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2025-02-17T19:02:11.386+0000] {processor.py:186} INFO - Started process (PID=5779) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:02:11.389+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:02:11.390+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:02:11.390+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:02:11.393+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:02:11.392+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:02:11.393+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:02:11.393+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:02:11.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:02:11.399+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-17T19:03:05.601+0000] {processor.py:186} INFO - Started process (PID=5845) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:03:05.604+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:03:05.605+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:03:05.605+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:03:05.607+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:03:05.607+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:03:05.608+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:03:05.607+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:03:05.608+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:03:05.614+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-17T19:13:01.374+0000] {processor.py:186} INFO - Started process (PID=7114) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:01.376+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:13:01.378+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:01.378+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:01.380+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:01.380+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:13:01.380+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:01.380+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:13:01.380+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:01.387+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-17T19:13:55.831+0000] {processor.py:186} INFO - Started process (PID=7337) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:55.833+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:13:55.835+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:55.834+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:55.837+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:55.836+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:13:55.837+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:13:55.837+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:13:55.837+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:13:55.843+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-17T19:14:52.354+0000] {processor.py:186} INFO - Started process (PID=7564) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:14:52.357+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:14:52.359+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:14:52.358+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:14:52.362+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:14:52.361+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:14:52.362+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:14:52.362+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:14:52.362+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:14:52.372+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-02-17T19:15:42.674+0000] {processor.py:186} INFO - Started process (PID=7767) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:15:42.676+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:15:42.678+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:15:42.677+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:15:42.680+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:15:42.680+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:15:42.681+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:15:42.681+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:15:42.681+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:15:42.687+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-17T19:19:46.294+0000] {processor.py:186} INFO - Started process (PID=8584) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:19:46.296+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:19:46.298+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:19:46.297+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:19:46.300+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:19:46.300+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:19:46.300+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:19:46.300+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:19:46.300+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:19:46.306+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2025-02-17T19:20:40.710+0000] {processor.py:186} INFO - Started process (PID=8811) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:20:40.711+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:20:40.714+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:20:40.713+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:20:40.718+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:20:40.718+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:20:40.718+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:20:40.718+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:20:40.719+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:20:40.728+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-02-17T19:21:36.155+0000] {processor.py:186} INFO - Started process (PID=9049) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:21:36.158+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:21:36.159+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:21:36.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:21:36.161+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:21:36.161+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:21:36.161+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:21:36.161+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:21:36.162+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:21:36.168+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-17T19:22:27.604+0000] {processor.py:186} INFO - Started process (PID=9260) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:22:27.607+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:22:27.608+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:22:27.608+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:22:27.611+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:22:27.610+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:22:27.611+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:22:27.611+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:22:27.611+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:22:27.617+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-17T19:23:18.928+0000] {processor.py:186} INFO - Started process (PID=9467) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:23:18.930+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:23:18.932+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:23:18.931+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:23:18.935+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:23:18.934+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:23:18.935+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:23:18.935+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:23:18.935+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:23:18.942+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-17T19:27:01.763+0000] {processor.py:186} INFO - Started process (PID=10246) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:27:01.765+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:27:01.767+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:27:01.767+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:27:01.775+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:27:01.774+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:27:01.775+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:27:01.775+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:27:01.776+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:27:01.786+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2025-02-17T19:41:53.151+0000] {processor.py:186} INFO - Started process (PID=14216) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:41:53.153+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:41:53.154+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:41:53.154+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:41:53.157+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:41:53.156+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:41:53.157+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:41:53.157+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:41:53.157+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:41:53.164+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-02-17T19:42:50.733+0000] {processor.py:186} INFO - Started process (PID=14452) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:42:50.735+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:42:50.737+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:42:50.737+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:42:50.740+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:42:50.740+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:42:50.741+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:42:50.740+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:42:50.741+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:42:50.750+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-02-17T19:43:47.597+0000] {processor.py:186} INFO - Started process (PID=14847) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:43:47.600+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:43:47.601+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:43:47.601+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:43:47.604+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:43:47.603+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:43:47.604+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:43:47.604+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:43:47.604+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:43:47.611+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-17T19:44:42.110+0000] {processor.py:186} INFO - Started process (PID=15224) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:44:42.113+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:44:42.117+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:44:42.116+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:44:42.123+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:44:42.122+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:44:42.124+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:44:42.124+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:44:42.124+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:44:42.148+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2025-02-17T19:45:31.650+0000] {processor.py:186} INFO - Started process (PID=15568) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:45:31.653+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:45:31.654+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:45:31.654+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:45:31.657+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:45:31.656+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:45:31.657+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:45:31.657+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:45:31.657+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:45:31.663+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-02-17T19:49:31.322+0000] {processor.py:186} INFO - Started process (PID=17173) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:49:31.325+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:49:31.327+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:49:31.326+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:49:31.329+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:49:31.329+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:49:31.329+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:49:31.329+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:49:31.330+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:49:31.339+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-02-17T19:50:28.465+0000] {processor.py:186} INFO - Started process (PID=17573) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:50:28.467+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:50:28.469+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:50:28.469+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:50:28.472+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:50:28.472+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:50:28.472+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:50:28.472+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:50:28.472+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:50:28.481+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2025-02-17T19:51:26.221+0000] {processor.py:186} INFO - Started process (PID=17969) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:51:26.223+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:51:26.225+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:51:26.224+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:51:26.228+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:51:26.227+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:51:26.228+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:51:26.228+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:51:26.228+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:51:26.237+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-02-17T19:52:19.758+0000] {processor.py:186} INFO - Started process (PID=18344) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:52:19.760+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:52:19.762+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:52:19.762+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:52:19.765+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:52:19.765+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:52:19.765+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:52:19.765+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:52:19.766+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:52:19.775+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2025-02-17T19:53:17.589+0000] {processor.py:186} INFO - Started process (PID=18746) to work on /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:53:17.592+0000] {processor.py:914} INFO - Processing file /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-02-17T19:53:17.593+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:53:17.593+0000] {dagbag.py:588} INFO - Filling up the DagBag from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:53:17.596+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:53:17.596+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-02-17T19:53:17.596+0000] {logging_mixin.py:190} INFO - [2025-02-17T19:53:17.596+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-02-17T19:53:17.597+0000] {processor.py:927} WARNING - No viable dags retrieved from /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-02-17T19:53:17.603+0000] {processor.py:208} INFO - Processing /home/aine/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
